### ISSUE TITLE
Make NFCPushOptions optional for pushMessage

### ISSUE THREAD

**Issue by alexshalamov** [w3c-cg/web-nfc#82]
In my opinion, by making NFCPushOptions optional, we can improve API usability and developers would need to type less code for main use-cases.

``` javascript
pushMessage(message);

// Instead of

pushMessage(message, {target: "tag", timeout: Infinity});
```

**Comment by zolkis** [comment]
If we can have sensible defaults, why not.
I expect that most of the pushes will be done to peers, and tags are much more often read than written.
If we need to set default values, then I am inclined to make "peer" the default target for a push.

Timeout could be set to Infinity as a default.

**Comment by zolkis** [comment]
We could also separate push (to peers) from write (to tags).
Then we could say

```
nfc.push(message1);
nfc.write(message2);
```

**Comment by zolkis** [comment]
Fixed by #88.
